15-lhcb-dev.md

```diff
@@ -76,16 +76,41 @@ It works similar to `lb-run`, without the need to specify a package and version:
 
 > ## What if getpack asks for my password 1000 times? {.callout}
 > `getpack` might ask you for your password several times.
-> To avoid this, you can create a kerberos token with
-> ```
-> kinit
-> ```
-> You will have to enter your password once, and further password prompts will be skipped
+> To avoid this, you need to configure your `ssh` following the
+> instructions [here](http://information-technology.web.cern.ch/book/how-start-working-svn/accessing-svn-repository#accessing-sshlinux).
+> The summary of the instructions follows:
+>
+>  1. Generate `ssh keys` (`ssh-keygen -t rsa` or `ssh-keygen -t dsa`).
+>  2. Copy your public `ssh` key to lxplus (`scp ~/.ssh/*.pub USERNAME@lxplus.cern.ch:~`).
+>  3. Execute the `ssh` setup script `/afs/cern.ch/project/svn/public/bin/set_ssh`.
+>  4. Setup your local `.ssh/config` file with
 > 
-> Alternatively, you can perform an anonymous checkout:
-> ```
-> getpack -p anonymous Phys/StrippingSelections
-> ```
+>     ```
+>     Host lxplus.cern.ch lxplus 
+>     Protocol 2 
+>     PubkeyAuthentication no 
+>     PasswordAuthentication yes
+>     
+>     Host svn.cern.ch svn 
+>     GSSAPIAuthentication yes 
```

Comment:
Should we add an explicit `PubkeyAuthentication yes`? I copied this
directly from the cern instructions...

On Thu, Dec 3, 2015 at 11:30 AM, roelaaij <notifications@github.com> wrote:

> In 15-lhcb-dev.md
> <https://github.com/lhcb/first-analysis-steps/pull/159#discussion_r46535177>
> :
>
> >  >
> > -> Alternatively, you can perform an anonymous checkout:
> > -> ```
> > -> getpack -p anonymous Phys/StrippingSelections
> > -> ```
> > +>     ```
> > +>     Host lxplus.cern.ch lxplus
> > +>     Protocol 2
> > +>     PubkeyAuthentication no
> > +>     PasswordAuthentication yes
> > +>
> > +>     Host svn.cern.ch svn
> > +>     GSSAPIAuthentication yes
>
> This doesn't make sense compared to the instructions. What you are doing
> here is allowing kerberos authentication with the SVN server, the public
> keys are actually not mentioned, and it depends on your global SSH
> configuration whether they are used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lhcb/first-analysis-steps/pull/159/files#r46535177>.
>



-- 
Dr. Albert Puig Navarro
Laboratoire de Physique des Hautes Energies
Ecole Polytechnique Fédérale de Lausanne (EPFL)
BSP 614.4 (Cubotron UNIL) CH-1015 Lausanne
EPFL Phone: 021 6939808
CERN Phone: 72518